sample/allofhands_hawaii2016/microlocations

```diff
@@ -0,0 +1,213 @@
+[
+    {
+        "id": 1, 
+        "name": "Convention Center Lobby", 
+        "floor": null, 
+        "latitude": "19.924029", 
+        "longitude": "-155.887498", 
+        "room": null
+    }, 
+   {
+        "id": 2, 
+        "name": "Convention Center", 
+        "floor": null, 
+        "latitude": "19.924029", 
+        "longitude": "-155.887498", 
```

Comment:
I'm not sure what you mean by "proper" coordinates. I have added the coordinated in all micro locations..